client/src/lib/components/ContactEmailsField/ContactEmailsField.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+  export let contactEmails: string[] = [""];
+  export let errors: string[] = [];
+
+  const add = () => {
+    contactEmails = contactEmails.concat("");
+    errors = errors.concat("");
+  };
+
+  const remove = (i: number) => {
+    contactEmails = contactEmails.filter((_, itemIndex) => {
+      return i !== itemIndex;
+    });
+    errors = errors.filter((_, itemIndex) => {
+      return i !== itemIndex;
+    });
+  };
+</script>
+
+<fieldset class="fr-fieldset fr-my-4w" aria-labelledby="contactEmails-legend">
+  <legend
+    class="fr-fieldset__legend fr-text--regular"
+    id="contactEmails-legend"
+  >
+    E-mail(s) de contact
+    <span class="fr-hint-text" id="contactEmails-desc-hint">
+      Vous pouvez ajouter des adresses e-mail personnelles en complément
+      l'dresse fonctionnelle. Ces emails seront régulièrement vérifiés afin
+      d'assurer la bonne maintenabilité des jeux de données.
+    </span>
+  </legend>
+
+  <ul class="fr-fieldset__content fr-raw-list  fr-mb-3w">
+    {#each contactEmails as _, i}
+      <li>
+        <div class="contact-entries">
+          <label for="contactEmails-{i}" hidden>Contact {i}</label>
+          <input
+            class="fr-input"
+            class:fr-input--error={errors[i]}
+            aria-describedby={errors[i]
+              ? `contactEmails-${i}-desc-error`
+              : null}
+            type="email"
+            id="contactEmails-{i}"
+            data-testid="contactEmails"
+            placeholder="email"
```

Comment:
Un placeholder plus "copieux" ?

```suggestion
            placeholder="Adresse e-mail"
```